chain/actors/builtin/init/init.go

```diff
@@ -0,0 +1,32 @@
+package init
+
+import (
+	"github.com/filecoin-project/go-bitfield"
+	"github.com/filecoin-project/go-state-types/cbor"
+	"golang.org/x/xerrors"
+
+	v0builtin "github.com/filecoin-project/specs-actors/actors/builtin"
+
+	"github.com/filecoin-project/lotus/chain/actors/adt"
+	"github.com/filecoin-project/lotus/chain/types"
+)
+
+func Load(store adt.Store, act *types.Actor) (State, error) {
```

Comment:
Unfortunately, is going to basically just be copied per actor. It could be done with reflection, but it's probably not worth it.